From 3cf363c7163e821e7736b70d03223ee07c8b4310 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 2 Jul 2023 06:21:47 -0400
Subject: [PATCH] Let native python interactive use the native loop

I think that this line is stopping me from using the bare python interpreter.

hard to say which is the right interaction, but this currently would fail on an import error due to missing ipython
---
 bindings/cython/datoviz/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bindings/cython/datoviz/__init__.py b/bindings/cython/datoviz/__init__.py
index 6d8c832f..54292c45 100644
--- a/bindings/cython/datoviz/__init__.py
+++ b/bindings/cython/datoviz/__init__.py
@@ -185,7 +185,7 @@ def run_native(n_frames=0, **kwargs):
 
 def run(n_frames=0, event_loop=None, **kwargs):
     event_loop = event_loop or 'native'
-    if event_loop == 'ipython' or is_interactive():
+    if event_loop == 'ipython':
         enable_ipython()
     elif event_loop == 'native':
         run_native(n_frames, **kwargs)
